### DISCUSSION TITLE
Noobie question - Getting my posts and the images?

### DISCUSSION THREAD

**Discussion by FrenchFryInteractive** [pocketbase/pocketbase#2555] [Q&A]
Hi, what is best approach for getting the posts and images of the current logged in user? I am thinking about using authstore and then pb functionality. I can get the titles and such, but the images are not coming along.

Is there a sure proof way to do it? It is in nextjs TP.

Thanks

**Comment by ganigeorgiev** [comment]
What do you mean by _"the images are not coming along"_?

We store only the filename as part of the Record model. If you want to download/access the uploaded file you can check https://pocketbase.io/docs/files-handling/#file-url.
